### ISSUE TITLE
With dask 2021.03, Coffea dask executor looks like doesn't work properly

### ISSUE THREAD

**Issue by oshadura** [CoffeaTeam/coffea-casa#86]
With dask 2021.03, Coffea dask executor looks like doesn't work:

```
fileset = {'SingleMu' : ["root://eospublic.cern.ch//eos/root-eos/benchmark/Run2012B_SingleMu.root"]}

from dask.distributed import Client
client = Client("tls://localhost:8786")

processor.NanoAODSchema.warn_missing_crossrefs = True

output = processor.run_uproot_job(fileset=fileset, 
                       treename="Events", 
                       processor_instance=Processor(),
                       executor=processor.dask_executor,
                       executor_args={'client': client, 'schema': processor.NanoAODSchema},
                       chunksize=250000)
```

Image: `coffea-casa/coffea-casa-analysis: 0.4.5`

**Comment by oshadura** [comment]
Error log:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-3df1df426421> in <module>
      9 processor.NanoAODSchema.warn_missing_crossrefs = True
     10 
---> 11 output = processor.run_uproot_job(fileset=fileset, 
     12                        treename="Events",
     13                        processor_instance=Processor(),

/opt/conda/lib/python3.8/site-packages/coffea/processor/executor.py in run_uproot_job(fileset, treename, processor_instance, executor, executor_args, pre_executor, pre_args, chunksize, maxchunks, metadata_cache)
   1135             }
   1136             pre_args.update(pre_arg_override)
-> 1137             pre_executor(to_get, metadata_fetcher, out, **pre_args)
   1138             while out:
   1139                 item = out.pop()

/opt/conda/lib/python3.8/site-packages/coffea/processor/executor.py in dask_executor(items, function, accumulator, **kwargs)
    729             ))
    730     else:
--> 731         work = client.map(
    732             function,
    733             items,

/opt/conda/lib/python3.8/site-packages/distributed/client.py in map(self, func, key, workers, retries, resources, priority, allow_other_workers, fifo_timeout, actor, actors, pure, batch_size, *iterables, **kwargs)
   1792             resources = None
   1793 
-> 1794         futures = self._graph_to_futures(
   1795             dsk,
   1796             keys,

/opt/conda/lib/python3.8/site-packages/distributed/client.py in _graph_to_futures(self, dsk, keys, restrictions, loose_restrictions, priority, user_priority, resources, retries, fifo_timeout, actors)
   2592                 retries = {k: retries for k in dsk}
   2593 
-> 2594             dsk = highlevelgraph_pack(dsk, self, keyset)
   2595 
   2596             # Create futures before sending graph (helps avoid contention)

/opt/conda/lib/python3.8/site-packages/distributed/protocol/highlevelgraph.py in highlevelgraph_pack(hlg, client, client_keys)
    113                 "__module__": None,
    114                 "__name__": None,
--> 115                 "state": _materialized_layer_pack(
    116                     layer,
    117                     hlg.get_all_external_keys(),

/opt/conda/lib/python3.8/site-packages/distributed/protocol/highlevelgraph.py in _materialized_layer_pack(layer, all_keys, known_key_dependencies, client, client_keys)
     63     }
     64 
---> 65     annotations = layer.pack_annotations()
     66     all_keys = all_keys.union(dsk)
     67     dsk = {stringify(k): stringify(v, exclusive=all_keys) for k, v in dsk.items()}

AttributeError: 'MaterializedLayer' object has no attribute 'pack_annotations'
```

**Comment by oshadura** [comment]
It could be a fault of the missing patch in oshadura/distributed.git :-( Rebuilding images, 
To avoid this problem in future see: https://github.com/CoffeaTeam/coffea-casa/issues/87

**Comment by oshadura** [comment]
Fixed in `coffea-casa:0.4.7` & `coffea-casa-analysis:0.4.7`
